### PR TITLE
BZ1999786: Added a KCS ref link for system reserved values

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.8+ versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1999786)
- [Preview](https://deploy-preview-40206--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-resources-configuring.html#nodes-nodes-resources-configuring-setting_nodes-nodes-resources-configuring)
- [4.7 PR](https://github.com/openshift/openshift-docs/pull/40207) 
- @sunilcio please verify.

### CODE REVIEW

File: modules/nodes-nodes-resources-configuring-setting.adoc [hunk 1]
```diff
@@ -26,41 +26,7 @@ For example, to access the resources from `cluster.node22` node, you can enter:
...
-                    "usageBytes": 129671168,
-                    "workingSetBytes": 102416384
-                },
-                "name": "runtime"
-            }
-        ]
-    }
-}
-----
+For more details, refer to link:https://access.redhat.com/solutions/5843241[Recommended system-reserved values]
```
Comment (bobfuru) [inline modules/nodes-nodes-resources-configuring-setting.adoc:29]:
```suggestion
For more details, refer to the link:https://access.redhat.com/solutions/5843241[recommended system-reserved values].
```

**Review by bobfuru — COMMENTED** [review]
A couple of minor clarifications and otherwise, this LGTM.

**Comment by Amrita42** [comment]
@bobfuru QE has approved . can be merged if all looks good.

**Pull request merged**